lib/jazzy/doc_builder.rb

```diff
@@ -329,17 +329,28 @@ def self.gh_token_url(item, source_module)
       "#{github_prefix}/#{relative_file_path}#{gh_line}"
     end
 
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/PerceivedComplexity
+    # rubocop:disable Metrics/MethodLength
+
     # Build mustache item for a top-level doc
     # @param [Hash] item Parsed doc child item
     # @param [Config] options Build options
     def self.render_item(item, source_module)
       # Combine abstract and discussion into abstract
       abstract = (item.abstract || '') + (item.discussion || '')
+      declaration = if Config.instance.hide_declarations == 'objc'
```

Comment:
Probably neater to pull these out into eg. `display_declaration` methods on `SourceDeclaration`.

Expected to see a similar change to `DocBuilder.document()` - doesn't the current code always show the objc declaration at the top of each page?

Reply:
done.

that behavior is actually on mustache templates. It doesn't show other declaration unless there is declaration. It picks up first declaration name from "language" and 2nd declaration is Swift by default. That's why I change the language at doc.rb to simplify the change.